test/e2e/liveness_probe_test.go

```diff
@@ -3,55 +3,19 @@
 package e2e
 
 import (
-	"context"
 	"fmt"
 	"strings"
 
 	"github.com/Azure/aad-pod-identity/test/e2e/framework"
 	"github.com/Azure/aad-pod-identity/test/e2e/framework/exec"
-	"github.com/Azure/aad-pod-identity/test/e2e/framework/mic"
 	"github.com/Azure/aad-pod-identity/test/e2e/framework/pod"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	corev1 "k8s.io/api/core/v1"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var _ = Describe("When liveness probe is enabled", func() {
 	It("should pass liveness probe test", func() {
-		pods := &corev1.PodList{}
-		Expect(kubeClient.List(context.TODO(), pods, client.InNamespace(framework.NamespaceKubeSystem))).Should(Succeed())
-
-		micPods := pod.List(pod.ListInput{
-			Lister:    kubeClient,
-			Namespace: framework.NamespaceKubeSystem,
-			Labels: map[string]string{
-				"app.kubernetes.io/component": "mic",
-			},
-		})
-
-		micLeader := mic.GetLeader(mic.GetLeaderInput{
-			Getter: kubeClient,
-		})
-
-		for _, micPod := range micPods.Items {
-			cmd := "clean-install wget"
-			_, err := exec.KubectlExec(kubeconfigPath, micPod.Name, framework.NamespaceKubeSystem, strings.Split(cmd, " "))
-			Expect(err).To(BeNil())
-
-			cmd = "wget http://127.0.0.1:8080/healthz -q -O -"
-			stdout, err := exec.KubectlExec(kubeconfigPath, micPod.Name, framework.NamespaceKubeSystem, strings.Split(cmd, " "))
-			Expect(err).To(BeNil())
-			if micPod.Name == micLeader.Name {
-				By(fmt.Sprintf("Ensuring that %s's health probe is active", micPod.Name))
-				Expect(strings.EqualFold(stdout, "Active")).To(BeTrue())
-			} else {
-				By(fmt.Sprintf("Ensuring that %s's health probe is not active", micPod.Name))
-				Expect(strings.EqualFold(stdout, "Not Active")).To(BeTrue())
-			}
-		}
-
```

Comment:
there is no more `clean-install` in MIC due to change in the base image.

Reply:
Not a blocker for this PR, but we should explore doing the health check with podIP from the busybox container which runs on host network